config/routes.rb

```diff
@@ -1,9 +1,9 @@
 Rails.application.routes.draw do
   resources :posts
   devise_for :users
+  
   root 'pages#index'
-  get '/pages' , to: 'pages#index'
```

Comment:
I see you removed this line altogether, I'd like to understand why so I can implement this in future code.

Reply:
For sure! Nothing 'wrong' with it per se. That line of code that I removed just ensured that somebody going to yegrb.com/pages/ goes to the home page.

I just left root path (just yegrb.com/ ) as the home page.

Really just a personal preference, eliminating the extra route! Hope that helps :)

Reply:
Yep, sure does.